### DISCUSSION TITLE
Welcome to Moksh Discussions!

### DISCUSSION THREAD

**Discussion by akshitagupta15june** [akshitagupta15june/Moksh#251] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dishal35** [comment]
I was assigned to redesign a button for the website, the button now has an img file inside of it, where should the img file be put while making a PR, since the img file is a local file

**Reply by harmeetsingh11** [reply]
You can add the local image to images folder. Then fetch it from there.
